lib/runtime/README.md

```diff
@@ -83,8 +83,7 @@ fn main() -> error::Result<()> {
     let mut instance = instantiate(WASM, &import_object)?;
 
     let values = instance
-        .func("add_one")?
```

Comment:
It's OK to just replace `func` by `dyn_func`. That's what I would recommend. Can you update the PR accordingly please?

Reply:
I'm sorry I was a little occupied in the last few weeks. Updating the PR ASAP.

Reply:
@ryzokuken awesome! Let us know once is ready to re-review :)